api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -259,8 +225,11 @@ public ResponseEntity<ReadOnlyNotebookResponse> previewExportToNotebook(
 
     List<String> codeCells = dataSetService.generateCodeCells(dataSetExportRequest, dbWorkspace);
 
+    KernelTypeEnum kernelType =
+        analysisLanguageMapper.analysisLanguageToKernelType(
+            dataSetExportRequest.getAnalysisLanguage());
     byte[] notebookHtml =
-        addCells(createNotebookObject(dataSetExportRequest.getKernelType()), codeCells)
+        addCells(createNotebookObject(kernelType), codeCells)
```

Comment:
This method is one place where `kernelType` continues to make sense, because it deals with the Jupyter kernel.